test/stdlib/StringBridge.swift

```diff
@@ -75,6 +56,45 @@ StringBridgeTests.test("Tagged NSString") {
 #endif // not 32bit
 }
 
+StringBridgeTests.test("Bridging") {
```

Comment:
This moved from tests/stdlib/SmallString.swift because this test depends on Foundation+ObjC but that file doesn't